studio/data/organizations/organization-billing-subscription-preview.ts

```diff
@@ -0,0 +1,80 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { post } from 'data/fetchers'
+import { organizationKeys } from './keys'
+import { ResponseError } from 'types'
+
+export type OrganizationBillingSubscriptionPreviewVariables = {
+  organizationSlug?: string
+  tier?: string
```

Comment:
shift the type `'tier_payg' | 'tier_pro' | 'tier_free' | 'tier_team' | 'tier_enterprise',` to here?